shell/browser/file_system_access/file_system_access_permission_context.h

```diff
@@ -102,6 +102,8 @@ class FileSystemAccessPermissionContext
   void NotifyEntryMoved(const url::Origin& origin,
                         const content::PathInfo& old_path,
                         const content::PathInfo& new_path) override;
+  void NotifyEntryRemoved(const url::Origin& origin,
```

Comment:
cc @codebytere a follow-up here, in case the in memory permission revoke should be mirrored to align with upstream feature.